contracts/core/PollingPlace.sol

```diff
@@ -233,10 +233,10 @@ contract PollingPlace is PollingPlaceInterface {
         BlockStoreInterface originBlockStore = BlockStoreInterface(_originBlockStore);
         BlockStoreInterface auxiliaryBlockStore = BlockStoreInterface(_auxiliaryBlockStore);
 
-        originCoreIdentifier = originBlockStore.getCoreIdentifier();
+        mosaicCoreIdentifier = originBlockStore.getCoreIdentifier();
         auxiliaryCoreIdentifier = auxiliaryBlockStore.getCoreIdentifier();
 
-        blockStores[originCoreIdentifier] = originBlockStore;
+        blockStores[mosaicCoreIdentifier] = originBlockStore;
```

Comment:
This should stay as origin core identifier. This is about the core identifiers on origin and auxiliary, not about "origin core". So `PollingPlace` should not change.

Reply:
Done